Annex60/Fluid/Movers/Data/SpeedControlled_Nrpm.mo

```diff
@@ -1,9 +1,18 @@
 within Annex60.Fluid.Movers.Data;
 record SpeedControlled_Nrpm "Generic data record for FlowMachine_Nrpm"
-  extends SpeedControlled_y;
+  extends SpeedControlled_y(
+    final constantSpeed = constantSpeed_rpm/speed_rpm_nominal,
+    final speeds =        speeds_rpm       /speed_rpm_nominal);
 
-  parameter Modelica.SIunits.Conversions.NonSIunits.AngularVelocity_rpm N_nominal = 1500
-    "Nominal rotational speed for flow characteristic";
+  parameter Modelica.SIunits.Conversions.NonSIunits.AngularVelocity_rpm constantSpeed_rpm=
+    speeds_rpm[size(speeds_rpm, 1)]
+    "Speed set point when using inputType = Annex60.Fluid.Types.InputType.Constant";
+
+  parameter Modelica.SIunits.Conversions.NonSIunits.AngularVelocity_rpm[:] speeds_rpm = {1500}
+    "Vector of speed set points when using inputType = Annex60.Fluid.Types.InputType.Stages";
+
+  final parameter Modelica.SIunits.Conversions.NonSIunits.AngularVelocity_rpm speed_rpm_nominal=
```

Comment:
I think it would make sense if this were not `final` since in some cases the highest speed curve does not allow to set up the `pressure` record, since data in the data sheet is incomplete due to a power limiting function. See for instance:
http://productfinder.wilo.com/repo/media/previews/00626996_0.jpg 
in this case it would be useful if the user would be able to override `speed_rpm_nominal`?